sources/utils.py

```diff
@@ -377,3 +377,18 @@ def split_list(l, nb_partitions):
     """
     return [l[i:(i+len(l)/nb_partitions)]
             for i in range(0, len(l), int(len(l)/nb_partitions))]
+
+def list_to_str(l):
```

Comment:
It's really not necessary. You can imagine to have a more generic function if you pass a "separator" parameter, with the value `-` by default.

```python
def list_to_str(seq, sep='-'):
    pass
```

Reply:
The `sep` argument has been added, however I'm not sure to understand your point, when you write "it's really not necessary". Do you mean that this function is useless?

Reply:
It's not necessary "for now", but I think it's a good design. See a comment above with the list of args and the `_` separator. We already have an good usage for this function with the `sep` parameter!